docs/source/installation/existing.md

```diff
@@ -140,7 +140,7 @@ default_images:
   jupyterlab: quansight/qhub-jupyterlab:v0.3.13
   dask_worker: quansight/qhub-dask-worker:v0.3.13
   dask_gateway: quansight/qhub-dask-gateway:v0.3.13
-  conda_store: quansight/qhub-conda-store:v0.3.13
+  conda_store: quansight/conda-store-server:v0.3.3
```

Comment:
Will Conda-store not follow the same version pattern as the other images?

Reply:
We will need to account for this in the upgrade command.